Local path example failed to run

### What happened?

When I try to run the example with the local path via 'make local', I get the error message on the Mac that the kcl file cannot be found.

### How can we reproduce it?

```
$ cd examples/resources/network
$ make local
crossplane render xr.yaml local_composition.yaml functions.yaml -r
crossplane: error: cannot render composite resource: pipeline step "normal" returned a fatal result: failed to run kcl function pipelines: Cannot find the kcl file, please check the file path /examples/resources/network/composition.k
make: *** [local] Error 1

```

### What environment did it happen in?
Function version: v0.10.0
KCL version: 0.10.0-darwin-arm64
kind cluster: 0.24.0
kubectl version
  Client Version: v1.31.0 
  Kustomize Version: v5.4.2
  Server Version: v1.31.0
uname -a: Darwin Kernel Version 24.0.0 root:xnu-11215.1.12~1/RELEASE_ARM64_T6020 arm64


<!--
Include at least the version or commit of Crossplane you were running. Consider
also including your:

* Cloud provider or hardware configuration
* Kubernetes version (use `kubectl version`)
* Kubernetes distribution (e.g. Tectonic, GKE, OpenShift)
* OS (e.g. from /etc/os-release)
* Kernel (e.g. `uname -a`)
-->

--- a/README.md
+++ b/README.md
@@ -122,6 +122,46 @@ spec:
   source: ./path/to/kcl/file.k
 ```
 
+> Note: You can't run the FileSystem example using `crossplane render` because it loads templates from a ConfigMap in the cluster. You can create a `ConfigMap` with the templates using the following command.
+
+```shell
+kubectl create configmap templates --from-file=templates.k -n crossplane-system
+```
+
+This `ConfigMap` will be mounted to the function pod and the templates will be available in the `/templates` directory. See the following function config for details.
+
+```yaml
+apiVersion: pkg.crossplane.io/v1beta1
+kind: Function
+metadata:
+  name: function-kcl
+spec:
+  package: xpkg.upbound.io/crossplane-contrib/function-kcl:latest
+  runtimeConfigRef:
+    name: mount-templates
+---
+apiVersion: pkg.crossplane.io/v1beta1
+kind: DeploymentRuntimeConfig
+metadata:
+  name: mount-templates
+spec:
+  deploymentTemplate:
+    spec:
+      selector: {}
+      template:
+        spec:
+          containers:
+            - name: package-runtime
+              volumeMounts:
+                - mountPath: /templates
+                  name: templates
+                  readOnly: true
+          volumes:
+            - name: templates
+              configMap:
+                name: templates
+```
+
 ### Read the Function Requests and Values through the `option` Function
 
 + Read the [`ObservedCompositeResource`](https://docs.crossplane.io/latest/concepts/composition-functions/#observed-state) from `option("params").oxr`.

